EIPS/eip-FOCIL.md

```diff
@@ -43,11 +43,16 @@ IL committee members construct their ILs and broadcast them over the P2P network
   By default, ILs are built by selecting raw transactions from the public mempool, ordered by priority fees, up to the IL’s maximum size in bits (e.g., 8 KB per IL). Additional rules can be optionally applied to maximize censorship resistance, such as prioritizing valid transactions that have been pending in the mempool the longest.
 
 #### Nodes
+- **`Slot N`, `t=0 to 9s`**: Nodes receive ILs from the P2P network and store (1) all new ILs that pass the CL P2P validation rules, and any evidence of IL equivocation by committee members (i.e., if multiple ILs are received from the same committee member).
 
-- **`Slot N`, `t=0 to 9s`**:
-Nodes receive ILs from the P2P network and only forward and cache those that pass the CL P2P validation rules.
+- **`Slot N`, `t=9s`**: IL freeze deadline. At this point, nodes freeze their IL view and stop caching new ILs in memory.
```

Comment:
We're using `IL freeze deadline` in the EIP but `VIEW_FREEZE_DEADLINE` in the consensus spec for the same purpose. Is it going to worth to use the same name to help readers understand?

Reply:
The reason I would call it view freeze deadline is that it might well end up being used for other things as well

Reply:
can it just be freeze deadline?

Reply:
changed in latest commit